src/libstd/rt/unwind/gcc.rs

```diff
@@ -129,7 +120,10 @@ pub mod eabi {
             uw::_URC_HANDLER_FOUND // catch!
         }
         else { // cleanup phase
-            uw::_URC_INSTALL_CONTEXT
+            unsafe {
+                __gcc_personality_v0(version, actions, exception_class, ue_header,
+                                     context)
+            }
```

Comment:
This is basically a roll-back of the rust_try_inner magic, which was created because of the way libgcc's C personality routine works on win64.  I thought it'd be good to get rid of unneeded complications. This is the way this code used to be a year ago.